File: src/EduApoyos.Application/Authentication/Models/AutenticacionResponse.cs
namespace EduApoyos.Application.Authentication.Models;

public class AutenticacionResponse
{
    public Guid UsuarioId { get; init; }

    public string NombreCompleto { get; init; } = string.Empty;

    public string Email { get; init; } = string.Empty;

    public string Rol { get; init; } = string.Empty;

    public string Token { get; init; } = string.Empty;

    public DateTime Expiracion { get; init; }
}

File: frontend/eduapoyos-web/src/app/core/auth/models/autenticacion-response.ts
export type RolUsuario =
  | 'Asesor'
  | 'Estudiante';

export interface AutenticacionResponse {
  usuarioId: string;
  estudianteId: string | null;
  nombreCompleto: string;
  email: string;
  rol: RolUsuario;
  token: string;
  expiracion: string;
}